lib/pure/asynchttpserver.nim

```diff
@@ -133,14 +133,21 @@ proc processClient(client: AsyncSocket, address: string,
     assert client != nil
     request.client = client
 
-    # First line - GET /path HTTP/1.1
-    lineFut.mget().setLen(0)
-    lineFut.clean()
-    await client.recvLineInto(lineFut) # TODO: Timeouts.
-    if lineFut.mget == "":
-      client.close()
-      return
+    # We should skip empty lines before the request
+    # https://tools.ietf.org/html/rfc7230#section-3.5
```

Comment:
The spec says that "at least one empty line" should be ignored. Let's remove this infinite loop.